dist/index.html

```diff
@@ -24,37 +24,62 @@
             <a href="#">News</a>
         </div>
     </div>
-    <div class="header-container">
-        <nav class="nav-wrapper">
-            <div class="mobile-nav">
-                <div class="hamburger">
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                </div>
-            </div>
-        </nav>
-        <div class="headline">
-            <h2>"Hello and Welcome!"</h2>
-            <h1>TECHEX </br> Blockchain-Expo 2023</h1>
-        </div>
-        <div class="headline-box">
-            <p>
-                This two day event will take place in November 2023, featuring 20
-                speakers and more than 500 participants. The theme for the two-day
-                Block-Chain Expo: Connecting the Blockchain Ecosystem
-            </p>
+    <div class="desktop-banner">
+        <div class="desktop-nav-banner">
+            <a href="#"><i class='fa fa-facebook'></i></a>
+            <a href="#"><i class="fa fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
         </div>
-        <div class="headline-date">
-            <h3>30 November ~ 1 December 2023</h3>
-            <p>@ Crystal Blue Tech Conference Hall</p>
+        <div class="desktop-nav">
+            <div class="banner-logo">
+                <a href="./index.html"><img src="./assets/logo/TechEx-Main-White-Small.png" alt="techexogo"></a>
+            </div>
+            <div>
+                <ul class="desktop-nav-list">
+                    <li> <a href="./about.html">About</a> </li>
+                    <li> <a href="#">Program</a> </li>
+                    <li> <a href="#">Join</a> </li>
+                    <li> <a href="#">Sponsor</a> </li>
+                    <li> <a href="#">News</a> </li>
+                </ul>
+            </div>
+            <div class="nav-campain">
+                <p>Our&nbsp;Campain</p>
+            </div>
         </div>
     </div>
```

Comment:
## HOME AND ABOUT PAGE

- Hey, to enforce good use of the semantic tag, you should use a nav tag here to wrap the nav items as the nav contains the navigation items in your webpage. let's kindly improve on this.

Reply:
fixed